src/main/java/com/github/manolo8/darkbot/core/manager/RepairManager.java

```diff
@@ -45,6 +54,10 @@ public void tick() {
 
         killerName = API.readMemoryString(API.readMemoryLong(repairAddress + 0x68));
         repairOptions.update(API.readMemoryLong(repairAddress + 0x58));
+
+        repairOptionsList.clear();
+        for (int i = 0; i < repairOptions.getSize(); i++)
+            this.repairOptionsList.add(repairOptions.get(i));
```

Comment:
should try to use `sync` in swf collection interface, it avoids having to re-make the whole list

Reply:
have to implement it